pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -95,7 +95,7 @@ func (e *cloudWatchExecutor) executeLogAction(ctx context.Context, model *simple
 		data, err = e.handleGetLogEvents(ctx, logsClient, model)
 	}
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to execute log action with subtype: %s: %w", subType, err)
```

Comment:
Example of where we have to use %w instead of %v.

Reply:
You can use the `errutil.Wrapf` package/function.